lib/rubocop/cop/metrics/method_length.rb

```diff
@@ -12,6 +12,9 @@ class MethodLength < Cop
         LABEL = 'Method'.freeze
 
         def on_def(node)
+          excluded_methods = cop_config['ExcludedMethods']
+          return if excluded_methods.include?(String(node.method_name))
```

Comment:
What does `method_name` return?

Reply:
I do not understand your question.
`node.method_name` return `Symbol`.